src/resources/authoring/gv2.json

```diff
@@ -408,7 +408,7 @@
       },
       {
         "comment": "*** FV Test Level 2 ***",
-        "challengeId": "gamete-1target-simple",
+        "challengeId": "egg-breeding-basic-match-visible",
```

Comment:
ok, I was hypothesizing that that might have been why the ITS wasn't working, but apparently that was a red herring.